src/python/library/tritonclient/http/_client.py

```diff
@@ -1254,7 +1254,7 @@ def generate_request_body(inputs,
             Indicates whether the request being added marks the end of the
             sequence. Default value is False. This argument is ignored if
             'sequence_id' is 0.
-        priority : int
+        priority : unsigned int
```

Comment:
python docstring-wise, I don't think there's a notion of an "unsigned int" in python, so this may be more confusing or harm static type checking tools, rather than being helpful. I'm not 100% sure if it would impact such tools though, just a hunch it would.